src/stories/PayPalMarks.stories.js

```diff
@@ -44,10 +44,31 @@ StandAlone.parameters = {
     },
 };
 
+const FormWithRadioButtons = (args) => {
+    const { fundingSources, onChange } = args;
+
+    return (
+        <form>
+            {fundingSources.map((source, index) => (
+                <label className="mark" key={index}>
+                    <input
+                        defaultChecked={index === 0 ? true : false}
```

Comment:
Small request, this line could be simplified.
```suggestion
                        defaultChecked={index === 0}
```

Reply:
Happy to make this change... but.... in the interest of surfacing useful code snippets is the ternary clearer ... we just don't know haha!